spec/game_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require_relative '../lib/game.rb'
+
+RSpec.describe Game do
+  let(:cell) { [[1, 2, 3], [4, 5, 6], [7, 8, 9]] }
+  let(:result) { Game.new(cell) }
+
+  describe '#result.position' do
+    it 'returns the result of the position of the element in the board' do
+      expect(result.cell).to eql(cell)
+    end
+  end
+
+  describe 'possible positon to win on the first row' do
+    let(:horizontal) { [%w[x x x], [4, 5, 6], [7, 8, 9]] }
+    let(:result_h) { Game.new(horizontal) }
+    it 'returns true when the first row has 3 equal symbols on the horizontal' do
+      expect(result_h.win_game).to eql(true)
+    end
+  end
+
+  describe 'possible positon to win on the second row' do
+    let(:horizontal) { [[1, 2, 3], %w[x x x], [7, 8, 9]] }
+    let(:result_h) { Game.new(horizontal) }
+    it 'returns true when the second row has 3 equal symbols on the horizontal' do
```

Comment:
- [ ] we have to test also the opposite cases for every test (what if there are no winning positions?)